Documentation/observability/metrics.rst

```diff
@@ -855,3 +888,30 @@ Options
 """""""
 
 This metric supports :ref:`Context Options<hubble_context_options>`.
+
+clustermesh-apiserver
+---------------------
+
+Configuration
+^^^^^^^^^^^^^
+
+To expose any metrics, invoke ``clustermesh-apiserver`` with the
+``--prometheus-serve-addr`` option. This option takes a ``IP:Port`` pair but
+passing an empty IP (e.g. ``:9962``) will bind the server to all available
+interfaces (there is usually only one in a container).
+
+Exported Metrics
+^^^^^^^^^^^^^^^^
+
+All metrics are exported under the ``cilium_`` Prometheus namespace.
+
+KVstore
+~~~~~~~
+
+======================================== ============================================ ========================================================
+Name                                     Labels                                       Description
+======================================== ============================================ ========================================================
+``kvstore_operations_duration_seconds``  ``action``, ``kind``, ``outcome``, ``scope`` Duration of kvstore operation
+``kvstore_events_queue_seconds``         ``action``, ``scope``                        Duration of seconds of time received event was blocked before it could be queued
```

Comment:
I understand the description was copied from above in the doc, but it looks like it could be improved? `Time received event was blocked before it could be queued, in seconds`?

ChatGPT suggesting a few others :) 

![image](https://github.com/cilium/cilium/assets/17001771/a242be02-2f9e-40b6-b8d5-0d0db10fac80)

Reply:
Yeah, that description seemed a bit weird also to me. I've tried to rephrase it in a separate commit to *"Seconds waited before a received event was queued"*. Does that sound better to you? If you prefer any other alternative I'll update it.